### ISSUE TITLE
external configuration problem GrailsMelody

### ISSUE THREAD

**Issue by GoogleCodeExporter** [javamelody/javamelody#163]
```
I'm having the following problem.

I wrote at the beginning of the file: GrailsMelodyConfig.groovy
grails.config.locations = ["file: / etc /.../${ appName}-config.groovy"]

In an attempt to change the configuration file using an external file.
But that I detected in GrailsMelodyConfig not have access to 
application.propreties

However in: Config.groovy and DataSource.groovy. The same command works

I'm using version 1.10 GrailsMelody.

What should I do to solve the problem?
```

Original issue reported on code.google.com by `ThiagoL...@gmail.com` on 30 Nov 2011 at 2:49
- Blocked on: #180
- Merged into: #180
